gradio/components/chatbot.py

```diff
@@ -88,14 +88,16 @@ class Message(GradioModel):
     metadata: Metadata = Field(default_factory=Metadata)
     content: Union[str, FileMessage, ComponentMessage]
 
+class ExampleMessage(TypedDict):
+    text: str
+    file: NotRequired[FileData]
```

Comment:
```suggestion
    files: NotRequired[list[FileData]]
```

Reply:
Sorry one more thing, why are we expecting users to provide FileData instead of just the path directly? FileData is mostly an implementation details that users should not worry about. So I actually think it should be like this^

```py
examples=[{"text": "Try this example with this audio.", "file": {"path": "files/cantina.wav"}}, {"text": "Try this example with this image.", "file": {"path": "files/avatar.png"}}, {"text": "This is just text, no files!"}],
```

cc @freddyaboulton as well since `FileMessage` has the same issue